### ISSUE TITLE
Can't type name in Create New Extension Class dialog

### ISSUE THREAD

**Issue by bjansen** [manifold-systems/manifold#108]
**Describe the bug**
When I add a new Extension Class, both input fields seem to be focused (hey both have a blinking cursor). When I type text, nothing happens. When I click inside the `Name` field, the text finally appears. Also, the `Esc` key doesn't work until I click inside this `Name` field.

**To Reproduce**
1. Create a new Extension Class
2. Type text without explicitly focusing the `Name` field

**Expected behavior**
Only the first field should be automatically focused, no need to use the mouse.

**Screenshots**
Both fields have a blinking cursor:

<img width="625" alt="image" src="https://user-images.githubusercontent.com/281528/61324221-3135d500-a812-11e9-8d1c-bc91f6ae3cfa.png">

**Desktop (please complete the following information):**
```
IntelliJ IDEA 2019.1.3 (Community Edition)
Build #IC-191.7479.19, built on May 28, 2019
JRE: 1.8.0_202-release-1483-b58 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
macOS 10.14.1
```
- Manifold version: 2019.1.7
- Manifold IntelliJ plugin version: 2019.1.7

**Comment by rsmckinney** [comment]
Hello again @bjansen.  There are some minor issues with that dialog I've been meaning to fix.  It incorporates UI components from the IJ API, but they don't play well together in the dialog. I'll see what I can do.  Thanks for reporting the issue!

**Comment by rsmckinney** [comment]
This appears to be fixed in IJ 2020.1
